lib/rspec/matchers/dsl.rb

```diff
@@ -336,7 +340,7 @@ class Matcher
         # Allows expectation expressions to be used in the match block.
         include RSpec::Matchers
 
-        # Converts matcher name and expected args to an English expresion.
+        # Facilitates better descriptions and failure messages.
         include RSpec::Matchers::Pretty
```

Comment:
I don't know.  It doesn't look easy.  `Pretty#name` is used in at least one place: `Matcher#inspect`.  

However, I think we can make `Pretty#underscore` private, and I have done so.